tenseal/cpp/tensors/encrypted_tensor_bfv.h

```diff
@@ -0,0 +1,263 @@
+#ifndef TENSEAL_TENSOR_ENCRYPTED_TENSOR_BFV_H
```

Comment:
Why do we need a separate interface from encrypted_tensor?

Reply:
No we don't sorry again, the wrong reason was the one mentioned above